src/modules/0.0.23/aws_vpc/mappers/route_table.ts

```diff
@@ -0,0 +1,278 @@
+import { EC2, paginateDescribeRouteTables } from '@aws-sdk/client-ec2';
+import { CreateRouteTableCommandInput } from '@aws-sdk/client-ec2/dist-types/commands/CreateRouteTableCommand';
+import {
+  Route as AwsRoute,
+  RouteTable as AwsRouteTable,
+} from '@aws-sdk/client-ec2/dist-types/models/models_1';
+
+import { AWS, crudBuilderFormat, paginateBuilder } from '../../../../services/aws_macros';
+import { getCloudId } from '../../../../services/cloud-id';
+import { findDiff } from '../../../../services/diff';
+import { Context, Crud2, MapperBase } from '../../../interfaces';
+import { Route, RouteTable, RouteTableAssociation } from '../entity';
+import { AwsVpcModule } from '../index';
+import { convertTagsForAws, convertTagsFromAws, eqTags } from './tags';
+
+export class RouteTableMapper extends MapperBase<RouteTable> {
+  module: AwsVpcModule;
+  entity = RouteTable;
+  equals = (a: RouteTable, b: RouteTable) => {
+    return (
+      a.vpc.vpcId === b.vpc.vpcId &&
+      this.eqListItems<Route>(this.eqRoute, a.routes, b.routes) &&
+      eqTags(a.tags, b.tags)
+    );
+  };
+
+  eqListItems<T>(eq: (a: T, b: T) => boolean, a?: T[], b?: T[]) {
+    const subset1 = !!a?.every(a1 => !!b?.find(a2 => eq(a1, a2)));
+    const subset2 = !!b?.every(a1 => !!a?.find(a2 => eq(a1, a2)));
+    return subset1 && subset2;
+  }
```

Comment:
should you check the length here too?

Reply:
I believe these are mathematically enough:
https://qr.ae/pr1NnC